src/styles/cards.scss

```diff
@@ -0,0 +1,164 @@
+.main {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  margin: auto;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 36px;
+    margin-bottom: 4px;
+    width: 166px;
+    transition: color, 300ms;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+    transition: transform 300ms;
+
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-left: 3px;
+  }
+
+  &__code {
+    color: $secondary;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 0;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    margin-bottom: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 166px;
+    height: 16px;
+  }
+
+  &__reviews-stars {
+    display: flex;
+    padding-top: 3px;
+    margin-left: 2px;
+  }
+
+  &__reviews-count {
+    color: $main-accent;
+    text-align: right;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    width: 166px;
+    height: 18px;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price-text {
+    color: $secondary;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price-amount {
+    color: $main-accent;
+    text-align: right;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 38px;
+    border: 1px solid $blue-accent;
+    border-radius: 5px;
+    background: $blue-accent;
+
+    color: $white;
+    text-align: center;
+    font-size: 14px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+
+  &__button:hover {
+    background-color: $white;
+    color: $blue-accent;
+    transition: color, 300ms;
+  }
+
+  .stars__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    &:not(:nth-child(n+5)) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
It should work as in the previous task, if you set modifier `stars--5` it should be 5 active stars

Reply:
I don’t see such a requirement either in the task or in the checkboxes.
Is it really needed?

Reply:
> I don’t see such a requirement either in the task or in the checkboxes. Is it really needed?

Sure, as your product could have 5 star rating or other